api/routes/ErrorLog.js

```diff
@@ -4,9 +4,8 @@ const ErrorLog = require('../models/ErrorLog');
 const { OK, BAD_REQUEST } = require('../constants').STATUS_CODES;
 const { addErrorLog } = require('../util/errorLog');
 
-router.post('/addErrorLog', async (req, res) => {
-  if (!await addErrorLog(req.body.userEmail, req.body.errorTime,
-    req.body.apiEndpoint, req.body.errorDescription)) {
+router.post('/addErrorLog', async (errorToAdd, res) => {
```

Comment:
just change the addErrorLog function param name, no need for anything else
```suggestion
router.post('/addErrorLog', async (req, res) => {
```